spec/javascripts/reducers/notification_spec.js

```diff
@@ -0,0 +1,44 @@
+import actionsTypes from "../../../app/assets/javascripts/actions/actionTypes";
+import reducer from "../../../app/assets/javascripts/reducers/notifications";
+
+describe("Notification Reducer",() => {
+  describe("DEFAULT", () => {
+    it("return the initialState", () => {
+      const initialState = [];
+      expect(reducer(undefined, initialState)).toEqual([]);
+    });
+  })
+
+describe('ADD_NOTIFICATION', () => {
+    it('adds a notification to the state',() => {
```

Comment:
We should have the two cases here, right? The one for a single notification and the other for multiple notifications.

Reply:
Done!